CMakeLists.txt

```diff
@@ -22,12 +22,18 @@ cmake_minimum_required(VERSION 3.0)
 project(MIVisionX)
 set(VERSION "2.2.0")
 
-#including GNUInstall Dir definitions.
-include( GNUInstallDirs )
+if(UNIX)
+  include(GNUInstallDirs CACHE)
+else()
+  set(${CMAKE_INSTALL_LIBDIR} "lib")
```

Comment:
I'm not certain about this at a high level, and at a technical level it's currently incorrect. This will dereference the variable `CMAKE_INSTALL_LIBDIR` and use it's value as a variable name to set. This should be `set(CMAKE_INSTALL_LIBDIR "lib")`, but I'm not even sure this is something we should be doing.

Reply:
@lawruble13 the Unix will include GNU Dirs in  Linux & macOS. What do we do in WIN32?